scipy/stats/stats.py

```diff
@@ -5981,6 +5982,14 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
 
         .. versionadded:: 1.6.0
 
+    trim : float, optional
+        Defines the percentage of trimming off of each end of the input
```

Comment:
```suggestion
        If nonzero, performs a trimmed (Yuen's) t-test.
        Defines the fraction of elements to be trimmed from each end of the input
```
I think the first line makes it more clear that a trimmed t-test is a different thing; it's not the same as removing data and performing a regular t-test.